common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -34,6 +34,17 @@ class @Problem
     @$('div.action input.reset').click @reset
     @$('div.action button.show').click @show
     @$('div.action input.save').click @save
+    # Accessibility helper for sighted keyboard users to show <clarification> tooltips on focus:
+    @$('.clarification').focus (ev) =>
+      icon = $(ev.target).children "i"
+      iconPos = icon.offset()
+      fakeEvent = jQuery.Event "mouseover", {
+        pageX: iconPos.left + icon.width()/2,
+        pageY: iconPos.top + icon.height()/2
+      }
+      icon.trigger(fakeEvent).trigger "click"
```

Comment:
Triggering fake events seems like the wrong solution here. The goal is to show the <clarification> tooltips on focus -- why not just use JS to, you know, show the tooltips? (Rather than triggering fake events that cause the tooltips to show themselves.)